.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "markdownlint.config": {
+        "MD028": false,
+        "MD025": {
+            "front_matter_title": ""
+        }
+    }
+}
```

Comment:
I suggest include the folder `.vscode/` as part of the `.gitignore` file to avoid that your specific setups will shared with the rest of the team